sqlglot/expressions.py

```diff
@@ -4793,6 +4793,11 @@ class List(Func):
     is_var_len_args = True
 
 
+# String pad, kind True -> LPAD, False -> RPAD
+class Pad(Func):
+    arg_types = {"kind": True, "this": True, "expression": True, "fill_pattern": False}
+
```

Comment:
Since we're using a bool flag I'd suggest renaming `kind` to something more descriptive, e.g. `lpad`, `is_lpad`, `left` are some ideas.

Reply:
kind should probably be one of the farther arguments since by default it would show up first